src/BPEssentials/Utils/ReloadUtil.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace BPEssentials.Utils
+{
+    public static class ReloadUtil
```

Comment:
I suggest changing this to `IO`, `File`, or `FileUtil`.
```suggestion
    public static class IO
```